lib/node_modules/@stdlib/repl/lib/help_text.js

```diff
@@ -71,6 +71,10 @@ var MSG = [
 	'    userDoc()               Add user-defined documentation.',
 	'    clearUserDocs()         Clear user-defined documentation.',
 	'',
+	'    settings()              List REPL settings.',
+	'    settings(name)          Get setting\'s value.',
+	'    settings(name,value)    Update setting with the new value.',
```

Comment:
```suggestion
	'    settings()              View and update settings.',
```

Nowhere in this help text do we split things out based on argument signatures, even though there are other variadic functions.

Reply:
I am missing something but `help` , `tutorial`, etc also have settings listed based on arguments right?

Reply:
Ah, you're right. I didn't expand the file to the top. I'd say to just document `settings()` generically and not list the various argument signatures. `help`, `tutorial`, `example` and `info` are effectively privileged as they are expected to be common commands where having call signatures broken out in more detail is warranted. I would not expect that `settings` will be common and we don't privilege the other commands listed with additional documentation for similar reasons.

If a user wants more detailed help text, we should provide such documentation when a user runs `help( settings )`.